### DISCUSSION TITLE
build failed on linux and windows

### DISCUSSION THREAD

**Discussion by tksoh** [eteran/nedit-ng#277] [Q&A] [answered]
@eteran the latest commit (4ae6a40c3450c76d4899e73b32af6a037e743c86) failed to compile:

> nedit-ng\src\DialogSmartIndentCommon.cpp(25): error C2039: 'setTabStopDistance': is not a member of 'QPlainTextEdit'

**Comment by tksoh** [comment]
The error message in my original post came from Windows.

The error on Linux looks slightly different:

> [ 40%] Building CXX object src/CMakeFiles/nedit-ng.dir/DialogMacros.cpp.o
> /home/tksoh/projects/nedit-ng/src/DialogMacros.cpp: In constructor ?DialogMacros::DialogMacros(QWidget*, Qt::WindowFlags)?:
> /home/tksoh/projects/nedit-ng/src/DialogMacros.cpp:30:16: error: ?class QPlainTextEdit? has no member named ?setTabStopDistance?; did you mean ?setTabStopWidth??
>   ui.editMacro->setTabStopDistance(tabStop * Font::characterWidth(ui.editMacro->fontMetrics(), QLatin1Char(' ')));
>                 ^~~~~~~~~~~~~~~~~~
>                 setTabStopWidth

**Comment by tksoh** [comment]
Looks like we need Qt 5.10 for this function?

> tabStopDistance : qreal
> This property holds the tab stop distance in pixels
> 
> By default, this property contains a value of 80.
> 
> This property was introduced in Qt 5.10.

**Comment by tksoh** [comment]
Replacing setTabStopDistance with setTabStopWidth (even though it's obsolete) appeared to fix the problem.

**Comment by tksoh** [comment]
@marilmanen

**Comment by eteran** [comment]
@tksoh Yup, my bad, easy fix though

**Comment by eteran** [comment, marked as answer]
@marilmanen and @tksoh Should be able to build now

**Comment by tksoh** [comment]
Looks good. Thanks.
